frontend/src/components/auth/ForgotPasswordEmailModal.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Box,
+  Image,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+import AuthModalBackground from "../../assets/forgotPasswordModalGraphic.svg";
+
+const ForgotPasswordEmailModal = ({
```

Comment:
nit: add documentation for this component

Reply:
Same as above regarding component documentation ^^